src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -396,8 +395,9 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       case Failure(e) =>
         logger.error(s"Preemption work failed for call ${callKey.tag}! " + e.getMessage, e)
         currentSender ! CallActor.Ack(message)
-        self ! CheckForWorkflowComplete
+        self ! CallFailedNonRetryable(callKey, events, returnCode, e.getMessage)
```

Comment:
This fails the call and doesn't leave it hanging "Running" if the Preemption work fails.